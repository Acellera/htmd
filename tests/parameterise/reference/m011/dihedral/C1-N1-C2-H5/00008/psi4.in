set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 0.894298	 -0.509540	 1.372311
C	 -0.119668	 -0.752603	 1.072087
N	 -0.528168	 0.128977	 -0.000000
H	 -0.763104	 -0.603587	 1.931634
H	 -0.142525	 -1.812763	 0.805996
C	 0.248044	 -0.037491	 -1.209632
H	 -1.498927	 -0.008356	 -0.199485
H	 0.998722	 0.743888	 -1.266824
H	 -0.398417	 0.061950	 -2.074060
H	 0.763621	 -0.999315	 -1.275517

	symmetry c1
}
ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "10  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
