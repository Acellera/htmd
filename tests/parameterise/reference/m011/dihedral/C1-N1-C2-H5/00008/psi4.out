    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 16:20:46 2016

    Process ID:  25067
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 0.894298	 -0.509540	 1.372311
C	 -0.119668	 -0.752603	 1.072087
N	 -0.528168	 0.128977	 -0.000000
H	 -0.763104	 -0.603587	 1.931634
H	 -0.142525	 -1.812763	 0.805996
C	 0.248044	 -0.037491	 -1.209632
H	 -1.498927	 -0.008356	 -0.199485
H	 0.998722	 0.743888	 -1.266824
H	 -0.398417	 0.061950	 -2.074060
H	 0.763621	 -0.999315	 -1.275517

	symmetry c1
}
ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "10  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-3-1-3.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 16:20:46 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           H          1.027525412010    -0.269243331560     1.424732699719     1.007825032070
           C          0.013559412010    -0.512306331560     1.124508699719    12.000000000000
           N         -0.394940587990     0.369273668440     0.052421699719    14.003074004780
           H         -0.629876587990    -0.363290331560     1.984055699719     1.007825032070
           H         -0.009297587990    -1.572466331560     0.858417699719     1.007825032070
           C          0.381271412010     0.202805668440    -1.157210300281    12.000000000000
           H         -1.365699587990     0.231940668440    -0.147063300281     1.007825032070
           H          1.131949412010     0.984184668440    -1.214402300281     1.007825032070
           H         -0.265189587990     0.302246668440    -2.021638300281     1.007825032070
           H          0.896848412010    -0.759018331560    -1.223095300281     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      1.18254  B =      0.31265  C =      0.27648 [cm^-1]
  Rotational constants: A =  35451.55665  B =   9372.88575  C =   8288.56087 [MHz]
  Nuclear repulsion =   84.660816602587389

  Charge       = 0
  Multiplicity = 1
  Electrons    = 26
  Nalpha       = 13
  Nbeta        = 13

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 32
    Number of basis function: 59
    Number of Cartesian functions: 59
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A         59      59       0       0       0       0
   -------------------------------------------------------
    Total      59      59      13      13      13       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 135
    Number of basis function: 418
    Number of Cartesian functions: 418
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 7.1258817386E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -109.53366658358040   -1.09534e+02   9.11911e-02 
   @DF-RHF iter   2:  -114.87188936734010   -5.33822e+00   5.92516e-02 DIIS
   @DF-RHF iter   3:  -131.75382987477792   -1.68819e+01   3.92906e-02 DIIS
   @DF-RHF iter   4:  -133.68302722548668   -1.92920e+00   1.77726e-02 DIIS
   @DF-RHF iter   5:  -134.16562152106610   -4.82594e-01   6.48764e-03 DIIS
   @DF-RHF iter   6:  -134.23379105222318   -6.81695e-02   7.73439e-04 DIIS
   @DF-RHF iter   7:  -134.23493778566296   -1.14673e-03   1.46842e-04 DIIS
   @DF-RHF iter   8:  -134.23500696637723   -6.91807e-05   3.72262e-05 DIIS
   @DF-RHF iter   9:  -134.23501132309968   -4.35672e-06   7.98617e-06 DIIS
   @DF-RHF iter  10:  -134.23501160120242   -2.78103e-07   2.29139e-06 DIIS
   @DF-RHF iter  11:  -134.23501161597159   -1.47692e-08   6.19069e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -15.536558     2A    -11.236010     3A    -11.235064  
       4A     -1.198982     5A     -0.947655     6A     -0.853684  
       7A     -0.661465     8A     -0.612492     9A     -0.590331  
      10A     -0.543437    11A     -0.509875    12A     -0.502867  
      13A     -0.359922  

    Virtual:                                                              

      14A      0.229091    15A      0.278053    16A      0.297516  
      17A      0.320937    18A      0.323817    19A      0.333496  
      20A      0.349854    21A      0.399592    22A      0.457978  
      23A      0.732729    24A      0.759997    25A      0.782269  
      26A      0.810009    27A      0.846657    28A      0.930806  
      29A      1.012430    30A      1.093406    31A      1.117686  
      32A      1.133233    33A      1.158842    34A      1.204722  
      35A      1.213168    36A      1.229648    37A      1.248156  
      38A      1.257142    39A      1.261838    40A      1.578033  
      41A      1.673600    42A      1.732896    43A      1.770488  
      44A      2.132861    45A      2.219704    46A      2.328227  
      47A      2.380234    48A      2.441662    49A      2.449388  
      50A      2.541102    51A      2.597281    52A      2.755375  
      53A      2.772980    54A      2.820805    55A      3.036164  
      56A      3.126210    57A      4.327263    58A      4.657038  
      59A      4.732277  

    Final Occupation by Irrep:
              A 
    DOCC [    13 ]

  Energy converged.

  @DF-RHF Final Energy:  -134.23501161597159

   => Energetics <=

    Nuclear Repulsion Energy =             84.6608166025873885
    One-Electron Energy =                -348.2150657140214207
    Two-Electron Energy =                 129.3192374954623460
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -134.2350116159717004

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     0.7382      Y:    -1.3563      Z:    -0.3179

  Electronic Dipole Moment: (a.u.)
     X:    -0.8876      Y:     0.9193      Z:     0.1531

  Dipole Moment: (a.u.)
     X:    -0.1494      Y:    -0.4371      Z:    -0.1648     Total:     0.4904

  Dipole Moment: (Debye)
     X:    -0.3797      Y:    -1.1109      Z:    -0.4190     Total:     1.2465


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-3-1-3.cx1.hpc.ic.ac.uk at Tue Jul  5 16:20:48 2016
Module time:
	user time   =       1.65 seconds =       0.03 minutes
	system time =       0.02 seconds =       0.00 minutes
	total time  =          2 seconds =       0.03 minutes
Total time:
	user time   =       1.65 seconds =       0.03 minutes
	system time =       0.02 seconds =       0.00 minutes
	total time  =          2 seconds =       0.03 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     0.7382      Y:    -1.3563      Z:    -0.3179

  Electronic Dipole Moment: (a.u.)
     X:    -0.8876      Y:     0.9193      Z:     0.1531

  Dipole Moment: (a.u.)
     X:    -0.1494      Y:    -0.4371      Z:    -0.1648     Total:     0.4904

  Dipole Moment: (Debye)
     X:    -0.3797      Y:    -1.1109      Z:    -0.4190     Total:     1.2465

  Quadrupole Moment: (Debye Ang)
    XX:   -19.0318     YY:   -22.7780     ZZ:   -19.8194
    XY:     0.4272     XZ:     0.3280     YZ:    -0.7426

  Traceless Quadrupole Moment: (Debye Ang)
    XX:     1.5112     YY:    -2.2349     ZZ:     0.7237
    XY:     0.4272     XZ:     0.3280     YZ:    -0.7426

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     H     0.41769  0.41769  0.00000  0.16462
       2     C     3.14550  3.14550  0.00000 -0.29099
       3     N     3.84032  3.84032  0.00000 -0.68064
       4     H     0.41762  0.41762  0.00000  0.16476
       5     H     0.43414  0.43414  0.00000  0.13173
       6     C     3.14488  3.14488  0.00000 -0.28977
       7     H     0.33146  0.33146  0.00000  0.33709
       8     H     0.41669  0.41669  0.00000  0.16663
       9     H     0.42220  0.42220  0.00000  0.15559
      10     H     0.42951  0.42951  0.00000  0.14097

   Total alpha = 13.00000, Total beta = 13.00000, Total charge =  0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
